modules/gitlab-runner/main.tf

```diff
@@ -0,0 +1,29 @@
+data "kubectl_path_documents" "values" {
```

Comment:
why we need kubectl, seems helm doing the job

Reply:
that is for adding our yaml files